VerilogCodingStyle.md

```diff
@@ -797,6 +797,70 @@ endmodule : foo
 localparam bit ValBaz = 1;  // This comment describes the item to the left.
 ```
 
+It can sometimes be useful to structure the code using header-style comments in
+order to separate different functional parts (like FSMs, the main datapath or
+registers) within a module. In that case, the preferred style is as follows:
+
+```systemverilog
+module foo;
+
+  ////////////////
+  // Controller //
+  ////////////////
```

Comment:
I think personally it is OK to also "standardize" on more simple forms of this, such as

```
//////////////
// Controller
```
If people feel like they have to follow this style to the letter, they'll hate the
need to count their slashes, even with guidance from their editor. MO

Reply:
Hi Scott.   I agree with hating to count anything, but isn't the original point of this all to squelch individuality in comments?  If we must have such guidelines, I personally would prefer them to be clear and easy to understand.   Perhaps make the style guide clear, and then as an organization discourage excessive nitpicky-ness in enforcement?

Reply:
I agree, let me add a couple of permissible variations.

Reply:
Actually, I think it is fine to keep this since it does say "header-style comments"
in "different functional parts". Not just any standard comment. Fine to standardize
here as much as possible.

Reply:
I am also with Martin here. I would prefer to agree on one single style.